active-sessions/src/client.rs

```diff
@@ -0,0 +1,30 @@
+use tonic::{transport::Channel, Request, Response, Status};
+use active_sessions::{UserData, UserDataResponse, ActiveSessions};
+
+pub mod active_sessions {
+    tonic::include_proto!("active-sessions");
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let channel = Channel::from_static("http://[::1]:50051")
```

Comment:
isn't port 50051 taken by auth-service?
Please see docker-compose.yml, this can lead to errors. 

**I think it's best to change this port**